pyms/DPA/PairwiseAlignment.py

```diff
@@ -598,7 +600,11 @@ def score_matrix_mpi(a1, a2, D):
 	return score_matrix
 
 
-def align_with_tree(T, min_peaks=1):
+class Pairwise(object):
+	pass
+
+
+def align_with_tree(T: Pairwise.Alignment, min_peaks=1) -> Alignment:
```

Comment:
This isn't right. `T` should be `pyms.DPA.PairwiseAlignment.PairwiseAlignment` and `class Pairwise` shouldn't be there.
Also `min_peaks` should be `int`.